packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -403,7 +403,7 @@ article > h1 {
 
   &:hover,
   &:focus {
-    background-color: $uswds-system-color-green-cool-vivid-70 !important;
+    background-color: $vads-button-color-background-primary-alt-active-on-light !important;
```

Comment:
I did a repo search for `$uswds` and `--uswds` to look for instances where uswds tokens may have been used, and this is the only one I found.